### ISSUE TITLE
salt-minion main process terminated with status 134

### ISSUE THREAD

**Issue by jcsp** [saltstack/salt#11710]
After running for some hours, the salt minion process has died.  This appears in dmesg output:

```
[28594.747901] init: salt-minion main process (1251) terminated with status 134
```

Salt version is salt-minion 2014.1.0-1precise1, running on Ubuntu 12.04.

No output in /var/log/salt/minion, no errors or warnings on the master log.

I have restarted the minions with logging set to debug.

**Comment by go8ose** [comment]
I noticed the LIBZMQ-39 bug is flagged as resolved, in various versions of zeromq. Are the libzmq3 packages provided by saltstack going to get rebuilt with a patch for this? On Debian Wheezy and Squeeze I can see libzmq3 is 3.2.3, the fixed version from zeromq is 3.2.5.

**Comment by basepi** [comment]
We should probably build more packages, but they will likely be version 4.1, as we've had success eliminating some transport issues with 4.1 over 3.x. Thanks for the bump.
